policyengine_us/parameters/gov/states/co/tax/income/rate.yaml

```diff
@@ -3,6 +3,8 @@ values:
   1999-01-01: 0.0475
   2000-01-01: 0.0463
   2020-01-01: 0.0455
+  2021-01-01: 0.045
+  2022-01-01: 0.044
```

Comment:
could you add references?

Reply:
The legislation is already referenced.

Reply:
I didn't see these values in the cited legal code.